src/components/Filters/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './styles.css';
+import { Switch, Route, Link } from 'react-router-dom';
+
+const Listar = () => <h1>Listar</h1>;
+const Adicionar = () => <h1>Adicionar</h1>;
+
+export function Filters() {
+  return (
+    <Switch>
+      <Route exact path="/listar">
```

Comment:
It's interesting 😮! As an English learner, I know it's difficult to write in another language 😭, but you don't feel alone. We are here to help you. 😄However, I would like to say some things:

1. I suggest using at least your code and PR only in English like `List` and `AddItem`. (Or in my humble opinion much better: `ListPage`, and `AddItemPage`, even you can create a `pages` folder)
2. Same line, your paths in your Routes: `/list` and `add-item`.

Bonus: Sometimes you need some help with an issue, it's easier to share your code other people but if they don't speak your native language, **it could be more difficult to get help.** 😄💚